app.ts

```diff
@@ -0,0 +1,106 @@
+﻿/// <reference path="jquery.d.ts" />
+
+/**
+ * Michael Leibbrandt 
+ * Typescript Onboard Project
+ */
+
+namespace onboard {
+
+	// Variable declarations
+
+	let totRec = getMax(); // Total amount of records
+	let totHeadings;
+	let fromRecord = 0;
+	let toRecord = 15;
+	let count = 0;
+	let increase = 15;
+
+	$(document).ready(function () {
+
+		// Creation of table as well as insertion of data
+
+		function populate(to, from) {
+
+			// Retrieves columns or Headings from API GET call
+
+			$.getJSON("/columns", function (data) {
+				if (count === 0) {
+					$('#btn2').prop("disabled", true);
+				} else if (count === totRec) {
+					$('#btn1').prop("disabled", true);
+				}
+				// Retrieves and sets column names
+
+				let headers = "<tr>";
+				for (let i = 0; i < data.length; i++) {
+					headers += `<th>${data[i]}</th>`
+				}
+				totHeadings = data.length;
+				headers += "</tr>";
+				$("#grid").append(headers);
+			});
+
+			let records = "<tr>";
+			if (to <= from) to = from;
+			$.getJSON("/records?from=" + from + "&to=" + to, function (data) {
+
+				// Loops throughout the data and inserts the records into the table
+
+				for (let i = 0; i < data.length; i++) {
+					for (let j = 0; j < totHeadings; j++) {
+						records += `<td>${data[i][j]}</td>`
+					}
+					records += "<tr/>";
+				}
+				$("#grid").append(records);
+			});
+		}
+
+		// Cleans up and retrieves previous table of data
+
+		$('#btn2').on('click', previous);
+
+		function previous() {
+			$('#btn1').prop("disabled", false);
+			$('#grid td').remove();
+			$('#grid tr').remove();
+			populate((toRecord - increase), (fromRecord - increase));
+			toRecord -= increase;
+			fromRecord -= increase;
+			count -= 1;
+			console.log(count);
```

Comment:
We usually remove all `console.log` calls that might have been used during development.  If you still want to log something you can use one of our `iq.log...` functions.